AxonDeepSeg/visualization/colorization.py

```diff
@@ -0,0 +1,102 @@
+# Tools to colorize instance maps with arbitrary color palettes
+
+import itertools
+
+import numpy as np
+import pandas as pd
+from skimage.future import graph
+from random import randint
+from PIL import Image, ImageDraw
+
+from loguru import logger
+
+
+class color_generator(object):
+    '''Generator that yields the next color given a color palette.'''
+
+    def __init__(self, colors, mem_length=50, tolerance=30):
```

Comment:
Would we maybe want to set an optional random.seed number somewhere in the class? In case we want to rerun it on a mask that's been manually editied slightly, maybe some users would want the same colors for each axons again?

Reply:
Hm yes this is an excellent idea. I'll look for a way to control the seed.

Reply:
Ok the random seed control is now implemented. I kinda want to keep the usage of this feature as it is (e.g. no value to pass when using the CLI, just adding the `-c` flag to activate colorization), but the seed control will be available for anyone using the API. I'll edit the morphometrics notebook to mention this. What do you think?

Reply:
>I kinda want to keep the usage of this feature as it is (e.g. no value to pass when using the CLI, just adding the -c flag to activate colorization), but the seed control will be available for anyone using the API. I'll edit the morphometrics notebook to mention this. What do you think?

I think this makes sense - agreed!